### ISSUE TITLE
Coffeelint 1.16.0 update, module.exports 'assigned to but never read' issue

### ISSUE THREAD

**Issue by bazineta** [za-creature/coffeescope#30]
After updating Coffeelint from 1.15.7 to 1.16.0, I'm getting 'Class is assigned to but never read' issues on anything that's assigned to a module.exports (we're a Webpack shop). Downgrading to 1.15.7 resolves the problem.
### Code sample

``` coffeescript
class Foo
# many exciting functions omitted
module.exports = Foo
```
### Output / stack trace

```
Class "Foo" is assigned to but never read
```
### coffeelint.json

``` json
"coffeelintConfig": {
    "arrow_spacing": {
      "level": "warn"
    },
    "braces_spacing": {
      "level": "ignore",
      "spaces": 0
    },
    "camel_case_classes": {
      "level": "error"
    },
    "coffeescript_error": {
      "level": "error"
    },
    "colon_assignment_spacing": {
      "level": "ignore",
      "spacing": {
        "left": 0,
        "right": 0
      }
    },
    "cyclomatic_complexity": {
      "value": 10,
      "level": "ignore"
    },
    "duplicate_key": {
      "level": "error"
    },
    "empty_constructor_needs_parens": {
      "level": "warn"
    },
    "ensure_comprehensions": {
      "level": "warn"
    },
    "eol_last": {
      "level": "warn"
    },
    "indentation": {
      "value": 2,
      "level": "error"
    },
    "line_endings": {
      "level": "warn",
      "value": "unix"
    },
    "max_line_length": {
      "level": "ignore"
    },
    "missing_fat_arrows": {
      "level": "ignore"
    },
    "newlines_after_classes": {
      "value": 3,
      "level": "ignore"
    },
    "no_backticks": {
      "level": "error"
    },
    "no_debugger": {
      "level": "warn"
    },
    "no_empty_functions": {
      "level": "ignore"
    },
    "no_empty_param_list": {
      "level": "ignore"
    },
    "no_implicit_braces": {
      "level": "ignore",
      "strict": true
    },
    "no_implicit_parens": {
      "strict": true,
      "level": "ignore"
    },
    "no_interpolation_in_single_quotes": {
      "level": "warn"
    },
    "no_plusplus": {
      "level": "ignore"
    },
    "no_stand_alone_at": {
      "level": "ignore"
    },
    "no_tabs": {
      "level": "error"
    },
    "no_this": {
      "level": "error"
    },
    "no_throwing_strings": {
      "level": "error"
    },
    "no_trailing_semicolons": {
      "level": "error"
    },
    "no_trailing_whitespace": {
      "level": "error",
      "allowed_in_comments": false,
      "allowed_in_empty_lines": true
    },
    "no_unnecessary_double_quotes": {
      "level": "warn"
    },
    "no_unnecessary_fat_arrows": {
      "level": "warn"
    },
    "non_empty_constructor_needs_parens": {
      "level": "ignore"
    },
    "prefer_english_operator": {
      "level": "warn",
      "doubleNotLevel": "ignore"
    },
    "space_operators": {
      "level": "warn"
    },
    "spacing_after_comma": {
      "level": "ignore"
    },
    "transform_messes_up_line_numbers": {
      "level": "warn"
    },
    "check_scope": {
      "module": "coffeescope2",
      "level": "warn",
      "environments": [
        "es5",
        "browser",
        "commonjs",
        "node"
      ],
      "overwrite": true,
      "shadow": false,
      "shadow_builtins": false,
      "shadow_exceptions": [
        "callback",
        "err",
        "next"
      ],
      "undefined": false,
      "hoist_local": true,
      "hoist_parent": true,
      "unused_variables": true,
      "unused_arguments": false
    },
    "internal_variable_names_usage": {
      "module": "coffeelint-internal-variable-names-usage",
      "level": "warn"
    },
    "no_implicit_returns": {
      "module": "coffeelint-no-implicit-returns"
    }
  },
```

**Comment by za-creature** [comment]
Dupe of #29

**Comment by za-creature** [comment]
Heads up in case you're not following the other issue: regression has been fixed in 0.4.3 but support for es6 imports / exports hasn't been added yet
